sw/device/rom_exts/rom_ext_common.ld

```diff
@@ -43,7 +22,14 @@ ENTRY(_rom_ext_start_boot)
  * s19->slm conversion scripts are not able to handle non-word aligned sections.
  */
 SECTIONS {
-  .rom_ext_manifest ORIGIN(eflash) : {
+  /**
+   * Phony section to create an offset for different slots.
+   */
+  .phony ORIGIN(eflash) : {
+    . = . + _slot_offset;
+  } > eflash
+
+  .rom_ext_manifest : {
```

Comment:
Instead of this, you can do the same by just setting the start of this section to a particular address, so you would have one of the following in the per-slot linker scripts:

```
_slot_start_address = ORIGIN(eflash);
```
or 
```
_slot_start_address = ORIGIN(eflash) + (LENGTH(eflash) / 2);
```

and then this in the linker script here:
```suggestion
  .rom_ext_manifest _slot_start_address : {
```

Reply:
This is an invalid syntax, but this `.rom_ext_manifest : ORIGIN(_slot_offset)` works fine.  Not sure how I missed it the first time around :)

Reply:
`_slot_offset` => `_slot_start_address`

Reply:
Done.